src/components/Navbar/index.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Nav, NavLink, Bars, NavMenu, NavBtn, NavBtnLink } from './NavbarElements';
+
+const Navbar = () => {
+    return (
+        <>
+            <Nav>
+                <NavLink to="/">
+                    <h1>InHabit</h1>
+                </NavLink>
+                <Bars />
+                <NavMenu>
+                    <NavLink to="/about_us" activeStyle>
+                        About
+                    </NavLink>
+                    <NavLink to="../templates/contact_us.html" activeStyle>
+                        Contact Us
+                    </NavLink>
+                    <NavLink to="/profile" activeStyle>
+                        Profile
+                    </NavLink>
+                </NavMenu>
+                <NavBtn>
+                    <NavBtnLink to="/log-in">Log Out</NavBtnLink>
+                </NavBtn>

```

Comment:
Do these `to="..."` work now or are they still mainly for visuals? If they do not work an idea I have is that it may be because the route of this Navbar file is in `/components/Navbar` rather than right in `src` with the App.js. So for them to work maybe we need to append `../../` or `../` to the front of each endpoint.